omrsigcompat/omrsig.cpp

```diff
@@ -333,11 +333,25 @@ omrsig_signalOS_internal(int signum, const struct sigaction *act, struct sigacti
 	if (NULL == signalOS) {
 		rc = -1;
 	} else {
-		sighandler_t old = signalOS(signum, act->sa_handler);
+		sighandler_t handler = SIG_DFL;
```

Comment:
Can you add a comment to this section to describe why we're defaulting to the SIG_DFL so that the next person looking at this understands?

Reply:
Comment added

Reply:
@DanHeidinga are you ok with the comment that was added?

Reply:
Yes - comment explains the "dance" going on here.